src/Login/hooks/usePostLoginToken.ts

```diff
@@ -0,0 +1,36 @@
+import { useMutation } from 'react-query';
+import { useNavigate } from 'react-router-dom';
+
+import { postLoginToken } from '../api/postLoginToken';
+
+const usePostLoginToken = () => {
+  const navigate = useNavigate();
+
+  const mutation = useMutation({
+    mutationFn: async (token: string) => {
+      return await postLoginToken(token);
+    },
+    onSuccess: (data) => {
+      const { tokenDto, nickname } = data;
+
+      if (nickname === null) {
```

Comment:
p5; 최근에 보고 나도 저렇게 써야지 했던 코드가 있었는데 마침 쓸 자리가 있길래 남겨봅니다 총총..

```suggestion
      if (!!nickname) {
```

!! : boolean 타입으로 명시적 형변환

Reply:
ASK: 헉 요거 그냥 이러케 해줘도 false로 넘어가지 않나용? 
```suggestion
      if (!nickname) {
```